### ISSUE TITLE
webP格式的图片无法下载

### ISSUE THREAD

**Issue by YinDongWu** [rs/SDWebImage#2458]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | . ios
 Platform Version        | 12
 SDWebImage Version      | 4.2.0 
 Integration Method     cocoapods
 Xcode Version           | Xcode 9
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by dreampiggy** [comment]
Duplicated for #2347 . And it's already been fixed in 4.4.1 version
